src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+
+  <body>
+    Hello, world!
```

Comment:
Wrap this text with any text tag (h1, h2...., p)

Reply:
use of any provided tags (h1, h2, h3 or p) fail `npm tests`.